tests/requirements.txt

```diff
@@ -1,4 +1,6 @@
--r requirements-test.txt
+pytest
```

Comment:
Would be nice to use tox instead of a separate requirements.txt file for tests.

Reply:
@GaretJax I was thinking of keeping it simple for now, want me to add tox already or is it okay to add later if necessary?

Reply:
Ok for now. There are no tests anyway.